### ISSUE TITLE
Consider unifying transport with air-transport and separating accommodation

### ISSUE THREAD

**Issue by jeisenbe** [gravitystorm/openstreetmap-carto#3631]
### Current situation:
- Accommodation icons (such as hotels and campsites) use the same blue icon color as transportation icons (such as bus stops, taxis, and parking lots): `#0092da`
- However, airports, helipads and ferry terminals have a violet color, `#8461C4`
- There are currently 4 or 5 shades of blue used for icons

The current code already has a color `@accommodation-icon` but this is set to be the same as `@transportation-icon` currently:
```
@transportation-icon: #0092da;
@transportation-text: #0066ff;
@accommodation-icon: @transportation-icon;
@accommodation-text: @transportation-text;
@airtransport: #8461C4; //also ferry_terminal
```

### Options:
1 Use  `#8461C4` (violet) for airtransport, transportation, and accommodations
![air-transport-for-transport-accomodation-tests](https://user-images.githubusercontent.com/42757252/50968066-5ec95b00-151d-11e9-90bb-d8d8e137745d.png)
- @imagico has shown this option on his _alt-colors_ fork, I believe.
- This reduces the number of shades of blue used for icons (currently 4 or 5, if springs count)

2. Use `#8461C4` (violet) for both airtransport and transportation-icon, while continuing to use `#0092da` (blue) for accommodation-icon
![accommodation-blue-all-transport-purple-test](https://user-images.githubusercontent.com/42757252/50977567-e91db900-1535-11e9-9428-f9344c551563.png)
- This is fairly logical; all transportation-related features use the same color
- But this does not address the problem of using too many shades of blue.

3. `#8461C4` (violet) for airtransport and transportation, new color for accommodations
![accomodation-a1395a-test](https://user-images.githubusercontent.com/42757252/50968223-be276b00-151d-11e9-855d-c5df57396eae.png)
- For example, `#a1395a` Lch(40,45,5)
- This color is similar to @tourism, but lighter and the hue is shifted slightly, to be less similar to `@shop` `#ac39ac` and `@health` `#BF0000`

**Comment by jeisenbe** [comment]
Here are some additional renderings with backgrounds:
1) `#8461C4` violet - for all
![air-transport-for-transport-accomodation-background](https://user-images.githubusercontent.com/42757252/50977809-65180100-1536-11e9-94b5-d87e5aafe10c.png)

2) `#8461C4` violet for all transportation, `#0092da` blue for accommodation
![accommodation-blue-all-transport-purple-backgrounds](https://user-images.githubusercontent.com/42757252/50977844-74974a00-1536-11e9-9197-1efd45c77fec.png)

3) `#8461C4` violet for transportation, `#a1395a` for accommodation
![accomodation-a1395a-backgrounds](https://user-images.githubusercontent.com/42757252/50977883-8d076480-1536-11e9-825a-64b235f38f15.png)

**Comment by kocio-pl** [comment]
For a bus station icon to change color you need to edit SVG code directly, probably in this line:

https://github.com/gravitystorm/openstreetmap-carto/blob/c59fb38794ff98fde263473efd48b5b72498c4da/symbols/amenity/bus_station.svg?short_path=ae19392#L86

**Comment by Tomasz-W** [comment]
I don't see any problems with current rendering accomodation objects in transportation-blue, I consider proposed change as the one which would produce more problems instead of resolving any one.

**Comment by jragusa** [comment]
This issue also raises (again ?) the debate about unification in transport amenities between `highway=bus_stop`, `railway=tram_stop/station` (#1127) and `amenity=ferry_terminal`. I think it will fix some blue icons.

**Comment by meased** [comment]
I like options 1 and 2.

Perhaps we could also discuss merging airtransport and transportation colors?

**Comment by polarbearing** [comment]
#3582 changed tourism outlines to ```#660033```, can we take that into the comparisons? I'd consider it (at least the hue) as a candidate for accommodation as well.

**Comment by kocio-pl** [comment]
I have difficulty saying if this is violet icon or brown. The only rescue are labels, which are different enough.

**Comment by jeisenbe** [comment]
@jragusa mentioned that railway=tram_stop/station uses yet another shade of blue: `@station-color: #7981b0;`

**Comment by Adamant36** [comment]
Things like motels that aren't transportation related per say could probably change to a different color. Things like bike parking and the parking symbol are better in blue though. Blue for transport is intuitive and a widely used color for those things. imagico's alternative color scheme just looks horrible. 

Even if blue is also used for water, I don't see why that would be an issue. Its widely different contexts and Know one is going to confuse a bus stop for a lake. 

Its worth consolidating the different shades of blue so there aren't so many, but that's about the only thing that's worth doing. Ultimately, I don't see a benefit to changing the color of transportation itself. Also, making hotels any shade of purple like in the last example puts them to close in color to things like bakeries. Which is just weird (although I notice bikes are the same color as bakeries for some reason). Maybe hotels could be changed to amenity brown (as its a service or something, I guess? Hotels aren't strictly related to transportation or tourism though is my point). Along with camp sites. 

Or campsites/alpine huts/caravan sites/whatever could be in man_made grey to fit shelters (or amenity brown along with hotels etc). 

As a side note, it looks as if fountain is in its own kind of blue shade maybe?

**Comment by jeisenbe** [comment]
>it looks as if fountain is in its own kind of blue shade maybe

Yes, there are 6 blue colors used for icons currently. 2 are transportation-related, 3 are water-related. 
1) transportation-icon: `#0092da`
2) office: `#4863A0` - used for the office dots
3) station-color: `#7981b0` - used for train stations and tram stops
4) marina-text: `#576ddf` - used for the fountain icon
5) water-text: `4d80b3` - used for the waterfall icon
6) The spring.svg file uses `#7abcec`

**Comment by polarbearing** [comment]
> Hotels aren't strictly related to ... tourism ... Along with camp sites.

No? I'd see those accommodations (hotel, motel, hostel, camp, huts) as the core of tourism. 

I agree that #660033 is a bit to close to amenity-brown, but #a1395a is too close to shop for me.

**Comment by jeisenbe** [comment]
Another option is to keep `#0092da` for transportation and air-transport. This would change airports, heliports and ferry terminals to `#0092da`. 
Accommodation could then use `#8461C4` (or another color.)

**Option 4:** `#0092da` for transportation and air-transport, `#8461C4` for accommodation
![test-0092da-transportation-8461c4-accommodation](https://user-images.githubusercontent.com/42757252/51066106-b960d500-164b-11e9-9490-df7ca08b9dfc.png)

![background-0092da-transportation-8461c4-accommodation](https://user-images.githubusercontent.com/42757252/51066107-ba920200-164b-11e9-9d5a-9c09c2a4e1a8.png)

**Comment by jeisenbe** [comment]
> I agree that #660033 is a bit to close to amenity-brown, but #a1395a is too close to shop for me.

I've tried adjusting the color a bit more. This new color was based off of the old health color, adjusted to be less similar to new heath and shop: `#ca4871` Lch(50,55,5)

**Option 5: `#ca4871` pink for accommodation, `#461C4` violet for transportation:**
![8461c4-transportation-ca4871-accommodation](https://user-images.githubusercontent.com/42757252/51066925-ed8ac480-1650-11e9-9a34-10c12282756f.png)

![8461c4-transportation-ca4871-accommodation-background](https://user-images.githubusercontent.com/42757252/51066929-f4193c00-1650-11e9-95f8-f1016de65c83.png)

**Comment by Adamant36** [comment]
@polarbearing > No? I'd see those accommodations (hotel, motel, hostel, camp, huts) as the core of tourism.

To me it depends. Hotels are used for a lot of reasons. Sometimes to visit family or go to a convention. They are also used by homeless people and for less savory purposes. I use to know someone who the manager of a pretty popular, although seeder, hotel and when the an ordnance passed banning people with local IDs from renting hotels to drive out the druggies they lost most of their business. 

I guess it depends on your definition of tourism though. Id say hotels are at least elastic enough to be considered in another category, if it allows for the color to be changed. I wouldn't necessarily go beyond that though.

**Comment by polarbearing** [comment]
Accommodation using previous airport #8461C4 seems good. 

The following test, #ca4871, goes too close to health.

**Comment by jeisenbe** [comment]
@polarbearing, what did you think about `#0db3170` in the first post?

**Comment by polarbearing** [comment]
Which post? The string '0db3170' appears the first time in this thread.

**Comment by jeisenbe** [comment]
> Which post? The string '0db3170' appears the first time in this thread

Sorry, that was a color I tested, but it seemed too saturated.
I meant **"Option 3.** in the first and second posts: `#a1395a` Lch(40,45,5)

**Comment by jeisenbe** [comment]
There is a large gap in hues of icons between blue and green currently, so it would be possible to create a new aquamarine / green-blue color such as `#07a1a3` (Lch60,35,138), for accommodation, transportation or another feature (craft?):

**Option 7: `#07a1a3` (Lch60,35,198) aqua:**

![8461c4-transportation-07a1a3-accommodation](https://user-images.githubusercontent.com/42757252/51164216-93536300-18df-11e9-8af7-94f2008c267f.png)

![8461c4-transportation-07a1a3-accommodation-background](https://user-images.githubusercontent.com/42757252/51164210-8fbfdc00-18df-11e9-9059-c6d20c93f8e7.png)

**Comment by jragusa** [comment]
@jeisenbe you forgot ferry routes `#6666ff`. Otherwise I agree to merge all water features into a unique colour

**Comment by jeisenbe** [comment]
I've opened issue #3641 to discuss the wider issue of picking new colors and improving the current range of colors for icons. There I've also suggested a third new color `#979506` in addition to the options above - but this would be better for landform-color than for transportation or accommodation icons.
